service/user.js

```diff
@@ -1,14 +1,18 @@
 const User = require("./schemas/user");
 const gravatar = require("gravatar");
+const { v4: uuidv4 } = require("uuid");
```

Comment:
dlaczego nie `nanoid`? 🤔

Reply:
myślałem, że to nie ma znaczenia, ale właśnie sobie z ciekawości wrzuciłem arytukuł nanoid vs uuid i już wiem dlaczego teraz tylko nanoid ;)